client/src/components/organisms/ChatContainer.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { Message } from "biseo-interface/chat";
```

Comment:
type 만을 import 하는 경우 `import type` 문법을 사용하는 것이 좋아요
runtime에서 사용되는 코드가 아니기에 typescript 컴파일 시에 삭제됩니당

또한 비서에서는 `biseo-interface`에 타입만을 의존하고 있기 때문에 `biseo-interface/~~~`에서 import 하는 경우 무조건 `import type`을 사용한다고 생각하면 될 것 같아요
```suggestion
import type { Message } from "biseo-interface/chat";
```